src/foam/nanos/auth/LifecycleState.js

```diff
@@ -27,7 +27,7 @@ foam.ENUM({
     },
     {
       name: 'DISABLED',
-      label: 'Disabled'
+      label: 'Paused'
```

Comment:
Please revert this.  If you need a label change, it needs to be app specific.

Reply:
reverted and [ticketed](https://nanopay.atlassian.net/browse/NP-8461)